### DISCUSSION TITLE
Sugestão de design do layout ( figma )

### DISCUSSION THREAD

**Discussion by gasampaiosouza** [BrasilAPI/BrasilAPI#226] [Show and tell]
## Português

olá ✋! eu fiz um pequeno design de como o site poderia ser ( pelo menos a parte da apresentação da API ).
**Fiquem a vontade** pra alterar e deixar o design no melhor estilo possivel.

> link do figma com o design
https://www.figma.com/file/toXC8uf52k7TGDAdJ2vXdd/brasilAPI?node-id=0%3A1

## English

Hey ✋! i made this design, kinda thinking of how the website's final design could be ( at least the API's presentation ).
**Feel free** to change the design, to let it be the best as possible!

> link on figma
https://www.figma.com/file/toXC8uf52k7TGDAdJ2vXdd/brasilAPI?node-id=0%3A1

**Comment by guiaramos** [comment]
Apenas um complemento:

Uma boa referencia de documentacao, seria da lib react-hook-form:  https://github.com/react-hook-form/documentation

Eles utilizam uma stack semelhante com a nossa, combinada com a praticidade do Gatsby e algumas libs bem promissoras:

- [React Simple Animate](https://react-simple-animate.now.sh/)
- [Little State Machine](https://github.com/bluebill1049/little-state-machine)

**Comment by enieber** [comment]
@gasampaiosouza ficou muito bom o layout, se vc puder fazer uma para a home, onde ficaria a logo uma explicação do projeto e os patrocinadores  do projeto. Caso não possa fazer tudo bem, o layout que vc fez já ficou muito bom 🚀 🚀

**Comment by RodriAndreotti** [comment]
> 
> 
> @gasampaiosouza ficou muito bom o layout, se vc puder fazer uma para a home, onde ficaria a logo uma explicação do projeto e os patrocinadores do projeto. Caso não possa fazer tudo bem, o layout que vc fez já ficou muito bom 🚀 🚀

Então, eu também não vejo isso como necessidade, na realidade tem até um PR já com uma documentação publicada com uma ferramenta que sugeri.
